apis/r/R/TileDBCreateOptions.R

```diff
@@ -229,9 +234,10 @@ TileDBCreateOptions <- R6::R6Class(
     #' @description ...
     #'
     #' @param build_filters Build filters into
-    #' \code{\link[tiledb:tiledb_filter-class]{tiledb_filter}} objects
+    #' \code{\link[tiledb:tiledb_filter-class]{tiledb_filter}} objects. If set to
+    #' \code{FALSE}, JSON strings are created instead of filter objects.
     #'
-    #' @return The create options as a list
+    #' @return The created options as a list
```

Comment:
I wrote this as "create options" because these are options for creation

Reply:
I see. Shall we put some ' around it then?

```suggestion
    #' @return The 'create options' as a list
```

Reply:
I like the quotes